.github/workflows/bin/generate-OPL-tables.pl

```diff
@@ -559,8 +565,8 @@ sub pgfiles {
 				}
 			} else {
 				# Tags are not valid, error printed by validation part.
-				print "File $name\n";
-				next;
+				#print "File $name\n";
+				return; # next is not valid for subs
```

Comment:
The other `next`s above should also be changed to `return`s.  This should also be fixed in the `webwork2/bin/OPL-update` file.  It must not happen generally for the problems in the OPL, or this would have been seen before.